scripts/hindent.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+hindent --validate src/**.hs test/*/**.hs
```

Comment:
in case we add subdirectories later on

```suggestion
hindent --validate src/*/**.hs test/*/**.hs
```

Reply:
When I run this in my bash I get this error
`hindent: /Users/lukasberglund/Code/hgb/src/*: getDirectoryContents:openDirStream: does not exist (No such file or directory)`

Reply:
Try `**/*.hs`

Reply:
It's not working for me. We could add this when we have subdirectories.

Reply:
sounds good